README.rst

```diff
@@ -435,13 +435,25 @@ it is not sent if the response was not successful.
 Default Alerts / Notifications
 ------------------------------
 
-+----------------------------+------------------------------------------------------------------------------------------------+
-|    Notification Types      | Uses                                                                                           |
-+----------------------------+------------------------------------------------------------------------------------------------+
-| ``threshold_crossed``      | Fires when a metric crosses the boundary defined in the threshold value of the alert settings. |
-+----------------------------+------------------------------------------------------------------------------------------------+
-| ``threshold_recovery``     | Fires when a metric goes back within the expected range.                                       |
-+----------------------------+------------------------------------------------------------------------------------------------+
++-------------------------------+------------------------------------------------------------------+
+| Notification Type             | Use                                                              |
++-------------------------------+------------------------------------------------------------------+
+| ``threshold_crossed``         | Used for creating notification when a metric exceeds             |
+|                               | threshold value.                                                 |
++-------------------------------+------------------------------------------------------------------+
+| ``under_threshold``           | Used for creating notification when a metric falls behind        |
+|                               | threshold value.                                                 |
++-------------------------------+------------------------------------------------------------------+
+| ``device_up``                 | Used for creating notification when a device gets connected.     |
++-------------------------------+------------------------------------------------------------------+
+| ``device_down``               | Used for creating notification when a device is unreachable.     |
++-------------------------------+------------------------------------------------------------------+
+| ``unreachable_after_upgrade`` | Used for creating notification when a device becomes unreachable |
+|                               | after firmware upgrade.                                          |
++-------------------------------+------------------------------------------------------------------+
+| ``connection_lost``           | Used for creating notification when the connection to a device   |
```

Comment:
Please let's have only two notification types:

- `connection_is_not_working`: used to notify when a connection (eg: SSH) to a device stops working (eg: credentials are outdated, management IP address is outdated, or device is not reachable)
- `connection_is_working`: used to notify when a connection to a device goes back to working normally

Include the failure reason in the notification text.

Don't mention the connection is lost, because it may not be lost.

Use "PROBLEM" and "RECOVERY" in email subjects.